res/nouns.txt

```diff
@@ -69,3 +69,7 @@ user directory
 cache
 arduino cluster
 neural network
+seg-fault
+seg-faults
```

Comment:
Conflicting entry; another noun exists with extremely similar meaning (I prefer this one over the other)

- "seg-fault"